src/components/tools/jobs.js

```
@@ -0,0 +1,68 @@
+        const progress = j.status && j.status === "completed" ?
+            "100" : j.progress && j.progress.percentage ?
+                Math.round(j.progress.percentage) : 0
+        tmpJobs.push({
+            rows: j.data.totalRows,
+            id: j.id,
+            user: j.data && j.data.user,
+            date: j.timestamp,
+            status: j.status,
+            waiting_time: j.processedOn && (j.processedOn - j.timestamp) / 1000,
+            processing_time: delay,
+            columns: validColumns.filter(c => j.data && j.data[c]),
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)